RavinduAye/Blogs
# develop branch updated

**# Blogs**

**# Clone the Repository**

```
git clone <repository_url>
cd <project_directory>
```

**# Client (Frontend)**

Update the conf.js file as follows:

```
 export default Object.freeze({
    appName : 'Blogs',
    server:{
        host: "http://blogs.test",
        redirectUri: "http://localhost:8080"
    }
})
```
cd client
npm install
npm run serve

**# Server (Backend)**

Update the .env file as follows:

```
DB_CONNECTION=mysql
DB_HOST=example.com
DB_PORT=3306
DB_DATABASE=mydatabase
DB_USERNAME=myuser
DB_PASSWORD=mypassword
```

composer install
php artisan key:generate
php artisan migrate
php artisan passport:install
php artisan serve

**Note** - **All the changes are in the develop branch.** checkout to develop before start the testing

Testing Steps

- The user is able to register to the system.
- The user is able to login to the system.
- Users are able to create, update, and delete posts. 
- Non-logged users can see the posts. They can not create, update, and delete posts.

### [blogs.docx](https://github.com/RavinduAye/Blogs/files/13061480/blogs.docx)


**# Additional Note**
- Due to time constraints, success messages have not been updated. 
- Console logs have been added for tracking purposes.
